pandas/tests/series/indexing/test_indexing.py

```diff
@@ -768,3 +768,19 @@ def test_head_tail(test_data):
     assert_series_equal(test_data.series.head(0), test_data.series[0:0])
     assert_series_equal(test_data.series.tail(), test_data.series[-5:])
     assert_series_equal(test_data.series.tail(0), test_data.series[0:0])
+
+
+def test_reset_index_drop_errmsg():
+    # https://github.com/pandas-dev/pandas/issues/20925
+
+    # Check KeyError raised for series where no 'level' name is defined
```

Comment:
series -> series index
no 'level' name is defined -> passed level name is missing

Reply:
done